pitop/robotics/json/alex.json

```diff
@@ -0,0 +1,44 @@
+{
+    "version": "0.17.0",
+    "classname": "Pitop",
+    "module": "pitop.system.pitop",
+    "components": {
+        "camera": {
+            "index": 0,
```

Comment:
I've noticed sometimes the camera will go onto another index and I've had to manually change it to get it working again. I'd recommend we change the USB camera implementation to check other indexes if zero fails so that it's more robust - of course, if somebody were to build a robot config with two cameras then the index of each would need to be specified on the fly (and they will probably change randomly between zero and one so putting this in a config might be troublesome anyway).

Reply:
that's right, the device index can change... I think we tried probing indexes once but it was just to slow (probing + waiting to fail if it was an incorrect index), but I guess we can do it again checking indexes from 0 to 3, or something like that.

Reply:
captured in https://github.com/pi-top/pi-top-Python-SDK/issues/268